modules/openapi-generator/src/main/resources/swift5/APIs.mustache

```diff
@@ -28,13 +27,16 @@ import Foundation
     /// If you need to get the request's progress in older OS versions, please use Alamofire http client.{{/useURLSession}}
     {{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}} var onProgressReady: ((Progress) -> ())?
 
-    required {{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}} init(method: String, URLString: String, parameters: [String:Any]?, isBody: Bool, headers: [String:String] = [:]) {
+    required {{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}} init(method: String, URLString: String, parameters: [String:Any]?, headers: [String:String] = [:]) {
         self.method = method
         self.URLString = URLString
         self.parameters = parameters
-        self.isBody = isBody
         self.headers = headers
 
+        if self.headers["Content-Type"] == nil {
+            self.headers["Content-Type"] = "application/json"
```

Comment:
I don't think we should default the content-type to `application/json` as we've reports before that will not work in certain use cases (in other clients)

Reply:
I copied that line from the kotlin client generator.
I will remove it.